src/meltano/core/tracking/environment.py

```diff
@@ -56,8 +56,7 @@ def __init__(self):
                 "meltano_version": meltano.__version__,
                 "is_dev_build": not release_marker_path.exists(),
                 "is_ci_environment": any(
-                    os.environ.get(marker, "").lower()[:1] in "1t"
-                    for marker in ci_markers
+                    os.environ.get(marker) for marker in ci_markers
```

Comment:
@pnadolny13 and @pandemicsyn - We could write (or import) a robust helpful function for 'truthy()' or we could assume no falsey values will be sent. Although I slightly prefer to check for a truthy value, I am okay with either approach since the edge cases are going to be relatively small volume, regardless.

Reply:
>I think what @pnadolny13 was saying is that if that env var is present its likely an indication of running in CI.

@pandemicsyn yep exactly

>just realized what the intention of that in '1t' check was. Its terse but not very "grokable" if you're just skimming the code, so might be worth rewriting if we keep it around.

@pandemicsyn ahhh that makes way more sense.

@aaronsteers yeah that looks to solve the problem I was seeing! Still not super easy to understand but it works. I'll accept your suggestion.